### DISCUSSION TITLE
Direct API access from postman or swagger

### DISCUSSION THREAD

**Discussion by pranay-garg-groww** [featurehub-io/featurehub#904] [Q&A] [answered]
Is there a postman collection or swagger of all the API's which I can directly test instead of integrating sdk?

**Comment by rvowles** [comment]
Hi there,

Could you tell us what you are trying to do and we could probably come up with some suggestions?

**Comment by pranay-garg-groww** [comment]
Basically I want to check responses of some API being called by SDK without integrating SDK. If we have a postman collection which I can import or there is swagger integration from where I can directly hit the API's?

**Reply by rvowles** [reply]
Hi again!

The reason I ask is I know what a postman collection is, we don't have anything like that, but Swagger appears to be a company that does a variety of things, so I'm not entirely sure what you are after there?

We do have OpenAPI definitions of all of the SDKs if that is any help? They are referred to in the documentation but they are used to generate the code for the ADKs.

**Reply by IrinaSouth** [reply]
hi @pranay-garg-groww , please find swagger files here https://docs.featurehub.io/featurehub/latest/admin-development-kit.html#_openapi_files

**Comment by rvowles** [comment, marked as answer]
As @IrinaSouth has indicated, our OpenAPI files are documented https://docs.featurehub.io/featurehub/latest/admin-development-kit.html#_openapi_files
